test/integration/features/user_creates_project.spec.js

```diff
@@ -7,15 +7,15 @@ const projectHelpers = require('../helpers/project.js');
 let goToAddToProjectPage = (projectName) => {
   let uri = 'project/' + projectName + '/create';
   h.goToPage(uri);
-  expect(element(by.cssContainingText('h1', "Create Using Your Code")).isPresent()).toBe(true);
-  expect(element(by.cssContainingText('h1', "Create Using a Template")).isPresent()).toBe(true);
+  expect(element(by.cssContainingText('.middle-container h1', "Create Using Your Code")).isPresent()).toBe(true);
```

Comment:
FYI @sg00dwin @rhamilto We'll need to update these tests in the vertical nav PR

Reply:
Done.  See https://github.com/spadgett/origin-web-console/pull/70/commits/ff58b2253223ef6c442456696d6d9155cc6a1fd7